build-scripts/build-rpm.sh

```diff
@@ -4,7 +4,9 @@
 source $(dirname "$(readlink -f "$0")")/build-srpm.sh
 
 # Install build dependencies
-dnf builddep -y rpmbuild/SRPMS/*src.rpm
+if [ x${SKIP_BUILDDEP} != "xyes" ]; then
```

Comment:
Isn't below more readable?

```
if [ "${SKIP_BUILDDEP}" != "yes" ]; then
```

Reply:
You're right. Without the quotes, `x` is required so that the left side is not empty if `SKIP_BUILDDEP` is not set. But, the quotes handle it.
Fixed